pkg/devfile/parser/content.go

```diff
@@ -0,0 +1,71 @@
+package parser
+
+import (
+	"bytes"
+	"io/ioutil"
+	"unicode"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+// Every JSON document starts with "{"
+var jsonPrefix = []byte("{")
+
+// YAMLToJSON converts a single YAML document into a JSON document
+// or returns an error. If the document appears to be JSON the
+// YAML decoding path is not used.
+func YAMLToJSON(data []byte) ([]byte, error) {
+
+	// Is already JSON
+	if hasJSONPrefix(data) {
```

Comment:
should we strip spaces before checking for `{`?

Reply:
it is being done in the `hasPrefix()` function.